apps/src/aichat/Aichat.tsx

```diff
@@ -0,0 +1,75 @@
+import PropTypes from 'prop-types';
+import React, {useRef, useLayoutEffect, useEffect, useState} from 'react';
+import styles from './aichat.module.scss';
+
+interface AichatProps {
+  children: React.ReactNode;
+}
+
+function useWindowSize() {
+  const [size, setSize] = useState([
+    document.documentElement.clientWidth,
+    document.documentElement.clientHeight,
+  ]);
+  useLayoutEffect(() => {
+    function updateSize() {
+      const width = document.documentElement.clientWidth;
+      const height = document.documentElement.clientHeight;
+      setSize([width, height]);
+    }
+    window.addEventListener('resize', updateSize);
+    updateSize();
+    return () => window.removeEventListener('resize', updateSize);
+  }, []);
+  return size;
+}
+
+/**
+ * Renders a simple AI chat.
+ */
+
+const Aichat: React.FunctionComponent<AichatProps> = ({children}) => {
+  // Leave a margin to the left and the right of the video, to the edges
```

Comment:
You probably won't need this layout stuff which is designed for a video which needs to maintain its aspect ratio while also filling the screen as much as possible.  A simple `flex` layout should be enough for your app to take up the whole page in a few panels.